changeset/spec/spec_helper.rb

```diff
@@ -2,6 +2,8 @@
 
 SPEC_ROOT = Pathname(__FILE__).dirname
 
+require SPEC_ROOT.join('../../spec/sql_overrides')
+
```

Comment:
I moved all the `rom-sql` overrides needed for testing to a single file. Once `rom-sq` has been adapted we just need to delete it and the lines requiring it.